exporter/datadogexporter/config_test.go

```diff
@@ -206,7 +206,7 @@ func TestUnmarshal(t *testing.T) {
 					},
 				},
 			}),
-			err: "1 error(s) decoding:\n\n* error decoding 'metrics.sums.cumulative_monotonic_mode': invalid cumulative monotonic sum mode \"invalid_mode\"",
+			err: "invalid cumulative monotonic sum mode \"invalid_mode\"",
```

Comment:
Actually, I believe it's still important to validate the field where the validation is failing. Is `metrics.sums.cumulative_monotonic_mode` still reported in the new version?

Reply:
It still is reported, adding that back now.